java/test/TestH5Sbasic.java

```diff
@@ -132,7 +134,7 @@ public void testH5Screate_simple_dims_invalid() throws Throwable
     public void testH5Screate_simple_dims_exceed() throws Throwable
     {
         long dims[] = {0,  1,  2,  3,  4,  5,  6,  7,  8,  9,  10, 11, 12, 13, 14, 15, 16, 17,
-                       18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 35};
+                       18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35};
```

Comment:
Doesn't hurt anything, but the new "34" marks 35 elements here for the call below, so "35" could just be removed.